src/audio/dai.c

```diff
@@ -797,6 +799,24 @@ static int dai_config(struct comp_dev *dev, struct sof_ipc_dai_config *config)
 				   channel);
 		break;
 	case SOF_DAI_IMX_SAI:
+		dd->config.burst_elems =
+			dd->dai->plat_data.fifo[dev->params.direction].depth;
+		break;
```

Comment:
Why are you modifying code for IMX_SAI in this commit? :)

Reply:
Apparently Github seems to show this as a diff rather than as preexisting code. It is already there in master.

Reply:
Got it.